src/test/scala/uk/gov/nationalarchives/tdr/api/utils/TestUtils.scala

```diff
@@ -111,9 +111,33 @@ object TestUtils {
   def addAntivirusMetadata(fileId: String): Unit = {
     val sql = s"insert into AVMetadata (FileId, Result, Datetime) VALUES (?, ?, ?)"
     val ps: PreparedStatement = DbConnection.db.source.createConnection().prepareStatement(sql)
-    ps.setString(1, s"${fileId}")
+    ps.setString(1, fileId)
     ps.setString(2, "Result of AVMetadata processing")
     ps.setTimestamp(3, Timestamp.from(FixedTimeSource.now))
     ps.executeUpdate()
   }
+
+  //scalastyle:off magic.number
```

Comment:
Why does the magic number styling gets turned off for this method but not for the others? Just wondering :slightly_smiling_face:

Reply:
When it is on, it doesn't allow you to add data to more than 4 (I think) fields, so I just needed it off for this one and another method in the "Client File Metadata" file.